### ISSUE TITLE
[Bug]: Getting Microsoft.JSInterop.JSException: 'Cannot read properties of null (reading 'innerText') when trying to show a dialog

### ISSUE THREAD

**Issue by Dodecadaemon** [fgilde/MudBlazor.Extensions#28]
### What happened?

I recently updated my project from v1.7.34 to v1.7.63 both are using MudBlazor 6.7.0, but the issue also happens in 6.2.0.

When I go to open any dialog, using the same code as I had before, I am getting the following error:

```
Microsoft.JSInterop.JSException: 'Cannot read properties of null (reading 'innerText')
TypeError: Cannot read properties of null (reading 'innerText')
    at MudExDialogHandler._updateDialog (https://localhost:7030/_content/MudBlazor.Extensions/MudBlazor.Extensions.lib.module.js:1:9236)
    at new MudExDialogHandlerBase (https://localhost:7030/_content/MudBlazor.Extensions/MudBlazor.Extensions.lib.module.js:1:7904)
    at new MudExDialogHandler (https://localhost:7030/_content/MudBlazor.Extensions/MudBlazor.Extensions.lib.module.js:1:12239)
    at new MudBlazorExtensionHelper (https://localhost:7030/_content/MudBlazor.Extensions/MudBlazor.Extensions.lib.module.js:1:18331)
    at Object.setNextDialogOptions (https://localhost:7030/_content/MudBlazor.Extensions/MudBlazor.Extensions.lib.module.js:1:18924)
    at https://localhost:7030/_framework/blazor.server.js:1:3501
    at new Promise (<anonymous>)
    at kt.beginInvokeJSFromDotNet (https://localhost:7030/_framework/blazor.server.js:1:3475)
    at https://localhost:7030/_framework/blazor.server.js:1:72077
    at Array.forEach (<anonymous>)'
```

I don't think I'm missing anything that would lead to this error, here is the method I use to show the popup:

```
public async void OpenFullScreenDialog<TDialogType>(
            DialogPosition destinationPosition,
            DialogParameters dialogParameters = null,
            Action<DialogResult> callback = null) where TDialogType : ComponentBase
    {
        var options = new DialogOptionsEx
            {
                MaximizeButton = false,
                CloseButton = false,
                NoHeader = true,
                Resizeable = false,
                Position = destinationPosition,
                Animations = new AnimationType[] { AnimationType.Slide },
                AnimationDuration = TimeSpan.FromMilliseconds(500),
                FullScreen = true,
                FullWidth = true,
                FullHeight = true,
                DisablePositionMargin = true,
                DisableSizeMarginY = true,
                DisableSizeMarginX = true
            };

        if (dialogParameters == null)
        {
            dialogParameters = new DialogParameters();
        }

        var dialog = await _dialogService.ShowEx<TDialogType>("Test", dialogParameters, options);
        if (callback != null)
        {
            callback(await dialog.Result);
        }
    }
```

And just in case it's something new on the dialog that I'm missing, here is one of the razor components that it's trying to show:

```
<MudDialog Class="mud-ex-dialog-initial">
    <DialogContent>
        <MudStack Class="pt-3" Justify="Justify.FlexStart" Row="true">
            <MudItem Class="pl-4">
                <MudIconButton Icon="@Icons.Material.Filled.Close" Variant="Variant.Outlined" Size="Size.Medium" OnClick="Close" />
            </MudItem>
            <MudText Class="pl-4" Typo="Typo.h4">Room Status</MudText>
        </MudStack>
        <MudContainer Style="width:50vw; overflow-y: scroll">
            <MudAlert Severity="Severity.Normal">The reactor type is RBMK-1000</MudAlert>
            <MudAlert Severity="Severity.Info">The reactor was fired up successfully</MudAlert>
            <MudAlert Severity="Severity.Success">The reactor is running at optimum temperature</MudAlert>
            <MudAlert Severity="Severity.Warning">The reactor temperature exceeds the optimal range</MudAlert>
            <MudAlert Severity="Severity.Error">Meltdown is imminent</MudAlert>
        </MudContainer>
    </DialogContent>
</MudDialog>


@code
{
    [CascadingParameter]
    MudDialogInstance MudDialog { get; set; }

    private void Close()
    {
        MudDialog.Close();
    }
}
```

### Expected Behavior

Would have expected the dialog to animate in like it does in v1.7.34

### What browser are you using?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fgilde** [comment]
Looks like there is a bug if 'NoHeader' is true.
I will fix it, until this is done you can temporary remove or disable the NoHeader option

**Comment by fgilde** [comment]
Bug is fixed in 1.7.64 but this version is currently not released, but preview is available in nuget. 

[1.7.64-d2307200948](https://www.nuget.org/packages/MudBlazor.Extensions/1.7.64-d2307200948)

Notice: Browser cache must be empty or it must be ensured that the MudBlazor.Extensions.lib.module.min.js has been updated or reloaded
